src/styles/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin-bottom: 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  width: var(--card-width-inside);
+  margin: 0 0 4px;
+  font-size: var(--font-size-normal);
+  font-weight: 500;
+  line-height: 1.5;
+  color: #060b35;
+}
+
+.card__product-code {
+  width: 98px;
+  margin: 0 0 16px;
+  font-size: var(--font-size-small);
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  width: var(--card-width-inside);
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.rating__reviews {
+  margin: 0;
+  font-size: var(--font-size-small);
+  font-weight: 400;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  width: var(--card-width-inside);
+  margin-bottom: 16px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__text-price {
+  font-size: var(--font-size-normal);
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__money {
+  font-size: var(--font-size-large);
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button-text {

```

Comment:
Why do you need this wrapper? just move this class to `a` tag and delete `button`